### DISCUSSION TITLE
Player Position Map Changes?

### DISCUSSION THREAD

**Discussion by dtcarls** [cwendt94/espn-api#419] [ESPN API]
Should we consider breaking down the various flex positions in the [player position map](https://github.com/cwendt94/espn-api/blob/997c4bb206cb74e9a9a87eab3df859830bfe3db8/espn_api/football/constant.py)?

For example, when a player is in the 'Flex' spot they are returned with `RB/WR/TE` but the Offensive Player Utility (OP) slot is returned with `OP` instead of `RB/WR/TE/QB`

The same can be said for the defensive player flexes as outlined on [this espn support page](https://support.espn.com/hc/en-us/articles/115003939192-Roster-Slots-Defense-). Specifically, Defensive Back (DB): can be `CB/S` and Defensive Player Utility (DP) can be `DT/DE/LB/CB/S`

**Comment by cwendt94** [comment]
Interesting, so are you thinking the flex spot would have like `Flex/RB` or would specify the players main position?

**Comment by dtcarls** [comment]
I found a work around for what I was trying to accomplish, just seems to be a small inconsistency between different Flex types. I found this while working with `player.slot_position` which leads to the examples listed above.

It has the potential to break some backwards compatibility if anyone is utilizing `player.slot_position` as currently implemented.
